### DISCUSSION TITLE
inline error in ardiuno version 2.0.1

### DISCUSSION THREAD

**Discussion by ghost** [pschatzmann/arduino-audio-tools#438] [Q&A] [answered]
what am I doing wrong ? or Maybe some missing configuration ?

c:\Users\ricardo\Documents\Arduino\libraries\arduino-audio-tools-main\src/AudioTools/AudioTypes.h:97:20: error: 'audio_tools::mime_pcm' declared as an 'inline' variable
 inline const char* mime_pcm = "audio/pcm"; ...... ETC.. ETC  ETC

thanks

**Comment by pschatzmann** [comment, marked as answer]
Inline variables are supported starting (C++17)

If you double check in the board manager you will see that you are still using an old ESP32 version: The latest is 2.0.5, so I suggest to upgrade.

In my latest audio tools version (I have just committed) you could also use #define USE_INLINE_VARS 0 to deactivate the global variables being defined inline.

**Reply by ghost** [reply]
thanks a lot ! ;   good job your libraries
regards
